### PR TITLE
[IE TESTS] ie_class migration to the new test infrastructure

### PR DESCRIPTION
**Pull request by iefode**

### CODE REVIEW

File: docs/template_plugin/tests/functional/shared_tests_instances/behavior/ie_class.cpp [hunk 1]
```diff
@@ -6,7 +6,11 @@
 #include <string>
 #include <vector>
 
-#include "ie_class.hpp"
+#include "behavior/ie_class.hpp"
```
Comment (ilya-lavrenov) [inline docs/template_plugin/tests/functional/shared_tests_instances/behavior/ie_class.cpp:9]: Can we rename to `core_integration.hpp`?

Comment (iefode) [reply docs/template_plugin/tests/functional/shared_tests_instances/behavior/ie_class.cpp:9]: yeah

Comment (iefode) [reply docs/template_plugin/tests/functional/shared_tests_instances/behavior/ie_class.cpp]: Done

File: inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp [hunk 2]
```diff
@@ -21,6 +21,9 @@ std::vector<std::string> disabledTestPatterns() {
 #if defined(_WIN32) || defined(_WIN64)
         //TODO: Issue: 33722
         ".*DynamicTranspose.*NonZero_Transpose.*",
+        // TODO: Issue 31197
+        R"(.*(IEClassBasicTestP).*smoke_registerPluginsXMLUnicodePath.*)",
 #endif
+        R"(.*IEClassLoadNetworkTest.*QueryNetworkHETEROWithMULTINoThrow_V10.*)",
```
Comment (gladilov-gleb) [inline inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp:27]: Is there a ticket for that?

Comment (iefode) [reply inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp:27]: Issue 34206

Comment (gladilov-gleb) [reply inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp:28]: Thanks!

**Comment by azhogov** [comment]
CI passed https://gitlab-icv.inn.intel.com/inference-engine/product-configs/merge_requests/574

**Pull request merged**